localstack/http/request.py

```diff
@@ -78,7 +81,17 @@ def form(self):
 
     @property
     def values(self):
-        raise NotImplementedError
+        """Naive implementation to parse and return form data.
+
+        The property is invoked by Moto ELB to get the ELB API version in
+        `moto.elb.urls.api_version_elb_backend()`
```

Comment:
I think we should keep the HTTP framework code free from references to specific services, I don't think this is relevant to the method here really. Although I appreciate the effort in documenting changes! :-)